content/en/Platform Deep Dive/Scans/faq.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "FAQ"
+linkTitle: "FAQ"
+weight: 70
+description: >
+  Frequently asked questions about the DAST scanner.
+---
+
+## How does the crawler work?
+The crawler starts with the target URL and any configured seed paths. It tries to follow and click all the links on a given page. The new pages and new injection points found are added to the list of URLs to scan. The crawler can detect patterns in the URLs and mark some URLs as duplicates. Some of the duplicates will be sampled to be scanned as well.
+
+## Can we scan an internal app?
+If the internal app can be accessed via an IP whitelist, we can test it. However, if a VPN/jumpbox or similar setup is required, we don't support that level of advanced configuration yet.
+
+## Why don't I receive email notifications?
+We send DAST emails to the organization owners only, not to the organization members. If you are an organization owner and still not receiving emails, please check your spam folder. If you still can't find the email, please contact us.
```

Comment:
• Nitpick: we use title case for user roles (e.g. Organization Owner)
• Here ('contact us') or elsewhere on the page, we could include the actual link to our DAST email